quickstarts/releaseiq/config.yml

```diff
@@ -0,0 +1,41 @@
+# Name of the quickstart (required)
+name: releaseiq
+
+# Displayed in the UI (required)
+title: ReleaseIQ
+
+# Long-form description of the quickstart (required)
+description: |
+  ## What's ReleaseIQ?
+  ReleaseIQ is the Unified Enterprise DevOps Platform to Accelerate Software Delivery. ReleaseIQ is focused on solving the problems of multi-tool, team, and product release pipelines.
```

Comment:
There are several typos, grammar issues and how the spelling of New Relic is incorrect, can you review this content and ensure this is corrected?